src/panoramic/cli/remote/writer.py

```diff
@@ -38,8 +49,22 @@ def delete(self, actionable: Actionable):
         else:
             raise NotImplementedError(f'delete not implemented for type {type(actionable)}')
 
-    def write(self, actionable: Actionable):
-        """Write data to remote api."""
+    def create(self, actionable: Actionable):
+        """Create data in the remote api."""
+        if isinstance(actionable, PanoField):
+            return self.create_field(actionable)
+        else:
+            self._upsert(actionable)
+
+    def update(self, actionable: Actionable):
+        """Update data in the remote api."""
+        if isinstance(actionable, PanoField):
+            return self.update_field(actionable)
+        else:
+            self._upsert(actionable)
+
+    def _upsert(self, actionable: Actionable):
+        """Upsert data to remote api."""
```

Comment:
I think we should use the upsert endpoint for both cases here just like with models  and datasets